timescaledb/load_data.py

```diff
@@ -0,0 +1,245 @@
+import json
+import psycopg2
+from pathlib import Path
+import xml.etree.ElementTree as et
+import os
+
+
+CONNECTION = "postgres://postgres:postgres@localhost:5433/test_db"
+BASEPATH = Path(__file__).parent
+TABLENAME = 'test_table'
+
+
+def create_table(conn):
+    """Create and install timescaledb plug-in if not exists.
+
+    Args:
+        conn (psycopg2.extensions.connection): The connection to PostgreSQL database.
+    
+    """
+    CREATE_SQL = """
+        CREATE TABLE %s (
+            id SERIAL,
+            time TIMESTAMPTZ,
```

Comment:
I think that django only support DateTime fields not TimeStampz so we might want to change that

Reply:
I don't think Postgres have a type called "DateTime". Since this is the SQL that we are going to execute within the database directly, I'll leave it as TIMESTAMPTZ here. Maybe we can use DateTime in django and Postgres can convert that to Timestamptz automatically.